lib/Bio/EnsEMBL/DataCheck/Checks/CompareHGNCMultipleGenes.pm

```diff
@@ -16,45 +16,65 @@ limitations under the License.
 
 =cut
 
-package Bio::EnsEMBL::DataCheck::Checks::HGNCMultipleGenes;
+package Bio::EnsEMBL::DataCheck::Checks::CompareHGNCMultipleGenes;
 
 use warnings;
 use strict;
 
 use Moose;
 use Test::More;
 use Bio::EnsEMBL::DataCheck::Test::DataCheck;
-use Bio::EnsEMBL::DataCheck::Utils qw/sql_count/;
 
 extends 'Bio::EnsEMBL::DataCheck::DbCheck';
 
 use constant {
-  NAME           => 'HGNCMultipleGenes',
-  DESCRIPTION    => 'HGNC-derived gene names are not given to multiple genes',
-  GROUPS         => ['core', 'xref', 'xref_name_projection'],
+  NAME           => 'CompareHGNCMultipleGenes',
+  DESCRIPTION    => 'HGNC-derived gene names assigned to multiple genes have not increased more than 5%',
+  GROUPS         => ['compare_core', 'xref', 'xref_name_projection'],
   DATACHECK_TYPE => 'advisory',
   TABLES         => ['external_db', 'gene', 'xref']
 };
 
 sub tests {
   my ($self) = @_;
 
-  my $dbea = $self->dba->get_adaptor('DBEntry');
+  my $cur_dba = $self->dba;
+  my $old_dba = $self->get_old_dba();
+
+  my $cur_multi_hgnc_percentage = $self->multiple_hgnc_percentage($cur_dba);
+  my $old_multi_hgnc_percentage = 0;
+
+  if (defined $old_dba) {
+    $old_multi_hgnc_percentage = $self->multiple_hgnc_percentage($old_dba);
+  }
+
+  my $multi_hgnc_diff = $cur_multi_hgnc_percentage - $old_multi_hgnc_percentage;
+
+  my $desc = "HGNC symbols assigned to multiple genes have not increased more than 5%";
```

Comment:
Do we expect HGNC symbol to increase by more than 5% every release?

Reply:
Genebuild suggested to fail the test when the proportion of HGNC symbols assigned to multiple genes increases more than 5% from the previous release.

Reply:
Was just wondering if a drop had the same potential issue.

Reply:
I did wonder the same thing, but I'm guessing it's not a problem to have several less symbols assigned to multiple genes. Also, this a very specific test and we have several other checks in place for HGNC and xrefs, so it's probably okay. I think we can ping Genebuild again and tune the threshold or add another check if we find that this one is too permissive.